Gaijinizer patch/Scripts.txt

```diff
@@ -3158,30 +3158,29 @@ world again. Until then, we look forward to your return.
 この結末の世界は、これっきりにしてほしい。
 =================================================="
 > CONTEXT: Scripts/テキスト出力/245:12
-"Subject: My mistake
- Sender [Richard]
+"Sender [Richard]
+Subject: My mistake
 
- 《Text》==========================================
-I misunderstood that you are the right person.
-If you dive in that 'game', you will pick it up
-If it was only 'that kind of person', he would wear a hawk.
+ 《Body》==========================================
+I thought you were the right person.
+I'd fooled myself into thinking that if I hid it in
+that 'game', that only a certain kind of person would
+choose to pick it up.
 
-But you got it.
-And I've played.
-That world is irrelevant to you, but it does exist.
- You made that world's destiny worse.
+But it was you picked it up and played.
+You might have no connection to that world, but it does
+exist, and you ruined its destiny.
 
-'It' has already been destroyed.
- It does not function as a 'game' or 'apparatus'.
- If you say that you have some trouble in that world
-I have no choice but to use a new 'device'.
-Or...... No, let's stop.
-This method is only negative for you.
+'That' has already been destroyed. It no longer functions
+either as a game or as a 'device'. If there's anything
+you wish to do, or undo, you'll have to use an entirely
+new 'device'. Or otherwise... no I'll stop there. That
+method has nothing but drawbacks for you.
```

Comment:
Minor flow:
```
Now, 'that' has already been destroyed.
It no longer functions either as a game or as a 'device'.
If there's anything you wish to do, or undo, you'll have to
use an entirely new one. 
Or otherwise... No, I'll stop there.
That method has nothing but drawbacks for you.
```